app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java

```diff
@@ -134,4 +145,42 @@ public void onClick(View v) {
         });
 
     }
+
+    @SuppressLint("ResourceType")
+    public void howToConnectDialog(String title, String intro, int iconID, String desc) {
+        try {
+            final AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            LayoutInflater inflater = getLayoutInflater();
+            View dialogView = inflater.inflate(R.layout.custom_dialog_box, null);
+            final SharedPreferences settings = this.getSharedPreferences(PREFS_NAME, 0);
```

Comment:
Use `Private` mode in sharedpreference and try to use a common `sharedPreferences` as this app is not too big to have multiple sharedpreferences.